### PR TITLE
Add funtion to get java path

### PR DESCRIPTION
**Pull request by orsolli**
I had problems running npm test, but the idea is that I can use the workspace specific java version so I don't have to change PATH every time I need to run a different java version

### CODE REVIEW

**Comment by orsolli** [comment]
Solves Issue #239

**Comment by orsolli** [comment]
#156

File: src/Utility.ts [hunk 1]
```diff
@@ -205,4 +205,13 @@ export namespace Utility {
...
         });
     }
+
+    export function getJavaExecutable(): string {
+        let javaPath = '';
+        const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('java');
+        if (config) {
+            javaPath = config.get<string>('home');
+        }
+        return javaPath + '/bin/java';
```
Comment (addy) [inline src/Utility.ts]: Is `'/bin/java'` the correct default here, or would you want to return `'java'` if `!config`?

Comment (orsolli) [reply src/Utility.ts]: You are absolutely right!

**Review by addy — APPROVED** [review]
LGTM

**Comment by andxu** [comment]
LGTM, will test it first.

**Review by andxu — APPROVED** [review]

**Pull request merged**